galvan-support/src/main/java/org/terracotta/testing/config/StartupBuilder.java

```diff
@@ -22,8 +22,9 @@
 
 public interface StartupBuilder {
   StartupBuilder tcConfig(Path tcConfig);
-  StartupBuilder serverInstallationDir(Path serverInstallationDir);
-  StartupBuilder testParentDir(Path testParentDir);
+  StartupBuilder serverKitDir(Path serverInstallationDir);
+  StartupBuilder serverLoggingExtension(String logging);
```

Comment:
`logging` could be renamed to `loggingExtension` or something.

Also, the name `loggingExtension` suffices for the API as it's all server side. The reason we had "server" prefix for working directory is that there's a unique working directory per server.